internal/message_queue/nats/client/nats_core/integration_test/client_integration_test.go

```diff
@@ -1,15 +1,15 @@
 package integration_test
 
 import (
+	natscore "github.com/bitcoin-sv/arc/internal/message_queue/nats/client/nats_core"
+	natsconnection "github.com/bitcoin-sv/arc/internal/message_queue/nats/nats_connection"
```

Comment:
No need for aliases as rule has been excluded

Reply:
Removed